tests/particle_output_multiple_formats/particles/particles-00000.0000.vtu

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" ?> 
```

Comment:
Please remove the .vtu files from the test output. The compressed output will likely change due to smallest floating-point changes, and we have no way of seeing whether it was a significant change or not. The .pvtu, .visit, and .pvd files should make clear that vtu output was written.